### ISSUE TITLE
Compile error - Swift project error - Could not generate Swift Representation

### ISSUE THREAD

**Issue by parikhparth23** [google/GTXiLib#27]
Please look at [ios-accessibility group conversation](https://groups.google.com/g/ios-accessibility/c/6Gn-kH4qjec) for more info

**Comment by CooperCorona** [comment]
I believe the 5.1.2 release of GTXiLib fixes the issue. It removes the C++ imports from the umbrella header, which broke Swift compatibility. Please try updating your Pods and building again. You may need to run `pod cache clean --all` or `pod repo update` to refresh the list of versions so CocoaPods uses the 5.1.2 release.

**Comment by parikhparth23** [comment]
thank you @CooperCorona - adding the version worked fine.

one question though - when i try to run this test https://github.com/parikhparth23/gtxilib_a11y/blob/main/a11yTests/a11yTests.swift i dont see any Accessibility error

**Comment by CooperCorona** [comment]
This is working as intended. The default SwiftUI project doesn't have any accessibility issues, so the tests pass. If you want to verify GTXiLib can catch issues, add
`Circle().accessibilityElement()` to your `VStack`. This will create a circle and make it an accessibility element. However, without any other accessibility information — such as an accessibility label — GTXiLib will mark it as a failure. If you added that line to your app, ran the app on a physical device, and turned on VoiceOver, you would be able to focus on the circle with VoiceOver, but it would not announce anything — an accessibility issue.

Try testing it with `ContentView` like:
```
VStack {
    Image(systemName: "globe")
        .imageScale(.large)
        .foregroundColor(.accentColor)
    Text("Hello, world!")
    Circle().accessibilityElement()
}
.padding()
```

**Comment by parikhparth23** [comment]
i added the `Circle().accessibilityElement()` to the `VSTACK` - https://github.com/parikhparth23/gtxilib_a11y/blob/main/a11y/ContentView.swift#L18

when i run on a physical device with the `voiceover` ON - it does not announce anything

But when i run the test - https://github.com/parikhparth23/gtxilib_a11y/blob/main/a11yTests/a11yTests.swift - GTXiLib is not marking it as a failure

**Comment by CooperCorona** [comment]
Ah, try renaming `override func setUpWithError() throws` to `override class func setUp()`. Note the `class` keyword. `setUpWithError` is an instance method that runs after the test suite has initialized but before each individual test case. Hence, `GTXTestSuite` can't install itself, because the original test suite has already been installed. Moving that code to the class version of `setUp`, which runs once per test suite before the test suite runs, lets GTXiLib install itself properly.
